simba/ui/machine_model_settings_ui.py

```diff
@@ -90,30 +90,78 @@ def __init__(self, config_path: str):
             "25",
             validation="numeric",
         )
-        self.estimators_entrybox.entry_set(val=2000)
```

Comment:
Thanks for doing this one.. remember you mentioned it but never got to it

Reply:
Maybe worth adding a default value just in case trouble reading from config? E.g.,         n_estimators = read_config_entry(self.config, ConfigKey.CREATE_ENSEMBLE_SETTINGS.value, MachineLearningMetaKeys.RF_ESTIMATORS.value, data_type=Dtypes.INT.value, default_value=2000)?

Reply:
Also, I realized if filling in the settings from the project_config, and sey it founds "random undersamle", then the random undersample ratio should also be filled in and the entry box has to be activated. Right now it is not activated: 
![image](https://github.com/sgoldenlab/simba/assets/34761092/01a2fea6-188e-4651-a103-2b3b5cbb35e8)